{{cookiecutter.project_slug}}/Dockerfile.node

```diff
@@ -0,0 +1,21 @@
+# pull official base image
+FROM node:12
+
+# set working directory
+WORKDIR /app
+
+# add `/app/node_modules/.bin` to $PATH
+ENV PATH /app/node_modules/.bin:$PATH
+
+# install app dependencies
+COPY package* ./
+# COPY package-lock.json ./
+RUN npm install --silent
+RUN npm install react-scripts@3.4.1 -g --silent
+
+# add app
+COPY public ./public
+COPY src ./src
+
+# start app
+CMD ["npm", "start"]
```

Comment:
Why would we need to run `npm start`? Won't the UI will always be served from the backend.

Reply:
This is for development use. Django only serves built React code and we don't want to run `npm run build` everytime there's a change when coding.

Reply:
yes, this file is meant for `docker-compose` usage in development
it builds the environment and installs dependencies, but then it maps the src directory to the image, so you can develop and the server will auto-refresh

`Dockerfile.heroku` uses `npm build` to build a production version and have Django serve it

@driverdan